app/models/location.rb

```diff
@@ -0,0 +1,15 @@
+class Location < ApplicationRecord
+  acts_as_mappable lat_column_name: :latitude, lng_column_name: :longitude
+
+  belongs_to :locatable, inverse_of: :location, polymorphic: true
```

Comment:
In plain English: "This model belongs to any model that `has_one` location defined as `locatable`"…

☝️ In our case, this behavior is mixed in to models as `include Locatable` (the concern) which itself includes into those models `has_one :location, as: :locatable, dependent: :destroy, inverse_of: :locatable`.